lib/ansible/modules/cloud/amazon/s3_sync.py

```diff
@@ -25,8 +25,8 @@
 short_description: Efficiently upload multiple files to S3
 description:
      - The S3 module is great, but it is very slow for a large volume of files- even a dozen will be noticeable. In addition to speed, it handles globbing,
-       inclusions/exclusions, mime types, expiration mapping, recursion, and smart directory mapping.
-version_added: "2.3"
+       inclusions/exclusions, mime types, expiration mapping, recursion, cache control and smart directory mapping.
+version_added: "2.4"
```

Comment:
This `version_added` is global for the module, not the last change to the module and should remain 2.3